app/build.gradle.kts

```diff
@@ -59,21 +59,27 @@ android {
         splits {
             // Configures multiple APKs based on ABI.
             abi {
-                // Enables building multiple APKs per ABI.
-                isEnable = true
+                val buildingAppBundle = gradle.startParameter.taskNames.any { it.contains("bundle") }
```

Comment:
A bit risky to use `taskNames.any`, because I think that some task(s) may fail if we run several tasks in the same gradle command. But I guess that here it's fine?

Reply:
The only problem could be if we run a gradle command like `./gradlew :app:assembleGplayRelease :app:bundleGplayRelease` I think, but that's probably not something we want to do at any point 😅 .

Reply:
yes. OK